frontend/src/Map/Map.tsx

```diff
@@ -1,43 +1,56 @@
-import React, { useEffect, useState } from 'react';
-import mapboxgl from 'mapbox-gl';
-import { DEFAULT_LONGITUDE, DEFAULT_LATITUDE, DEFAULT_ZOOM } from 'constants/constants';
-import './Map.css';
+import React, { useEffect, useState } from "react";
+import mapboxgl from "mapbox-gl";
+import {
+  DEFAULT_LONGITUDE,
+  DEFAULT_LATITUDE,
+  DEFAULT_ZOOM,
+} from "constants/constants";
+import "./Map.css";
 
-mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_TOKEN || '';
+mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_TOKEN || "";
 
 interface IMapProps {
-    mapContainer: HTMLElement | string | null;
+  mapContainer: HTMLElement | string | null;
+  murals: any;
 }
 
 function Map(props: IMapProps) {
```

Comment:
we can "spread" the props by writing `function Map({mapContainer, murals} as IMapProps)`